ci/ansible/all.yml

```diff
@@ -36,7 +36,8 @@ ovn_number_chassis: 5
 ########################
 network_start_cidr: "172.16.201.0/24"
 network_number: "5"
-ports_per_network: "50"
+ports_per_network: "500"
```

Comment:
Do we really want to default this to 500? We should allow for this to be configured on the commandline, or have our downstream CI just modify all.yml before running.